examples/README.md

```diff
@@ -9,17 +9,10 @@
 
 ### Use ClimaCore Examples from the ClimaCore parent package
 
-    cd ClimaCore
-    julia --project=examples
-
-Activate the Pkg repl mode and dev the ClimaCore repo and ClimaCorePlots packages into the examples project environment:
-
-    (examples) pkg> dev . lib/ClimaCorePlots
-    (examples) pkg> instantiate
+Instantiate the examples environment (this will download and install the necessary packages)
 
-Or alternatively:
-
-    julia --project=examples -e 'using Pkg; Pkg.develop([Pkg.PackageSpec(path="."), Pkg.PackageSpec(path="lib/ClimaCorePlots")])'
+    cd ClimaCore
```

Comment:
I would leave a note in here that the manifest `dev` the lib/ folder sub-projects, otherwise it could be confusing